lib/jekyll-remote-theme/downloader.rb

```diff
@@ -38,16 +47,20 @@ def downloaded?
       private
 
       def zip_file
-        @zip_file ||= Tempfile.new([TEMP_PREFIX, ".zip"])
+        @zip_file ||= Tempfile.new([TEMP_PREFIX, ".zip"], :binmode => true)
       end
 
       def download
         Jekyll.logger.debug LOG_KEY, "Downloading #{zip_url} to #{zip_file.path}"
-        request = Typhoeus::Request.new zip_url, TYPHOEUS_OPTIONS
-        request.on_headers  { |response| raise_if_unsuccessful(response) }
-        request.on_body     { |chunk| zip_file.write(chunk) }
-        request.on_complete { |response| raise_if_unsuccessful(response) }
-        request.run
+        io = URI(zip_url).open(OPTIONS)
+        IO.copy_stream io, zip_file
+        OpenURI::Meta.init zip_file, io
```

Comment:
Hm, I couldn't find this method in Dash from the ruby stdlib, and rubydoc.info didn't have any docs for the method I did find. What does this do?

Reply:
Had the same question... https://apidock.com/ruby/OpenURI/Meta/init/class